test/extended/authorization/rbac/groups_default_rules.go

```diff
@@ -182,47 +183,60 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] The default clust
 
 	oc := exutil.NewCLI("default-rbac-policy")
 
-	g.It("should have correct RBAC rules", func() {
-		kubeInformers := informers.NewSharedInformerFactory(oc.AdminKubeClient(), 20*time.Minute)
-		ruleResolver := exutil.NewRuleResolver(kubeInformers.Rbac().V1()) // signal what informers we want to use early
+	g.It(
+		fmt.Sprintf("should have correct RBAC rules [apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s][apigroup:%s]",
+			buildGroup,
+			userGroup,
+			projectGroup,
+			authzGroup,
+			security.GroupName,
+			consoleGroup,
+			"helm.openshift.io",
+			templateGroup,
+			operatorsCoreOSGroup,
+			imageGroup,
+		),
+		func() {
```

Comment:
Would you please put `func() {` right after the previous line as `(, func() {`? To reduce the diff. If that does not help you might move `fmt.Sprintf` next to `g.It(`.

Reply:
Done